test-case/check-alsabat.sh

```diff
@@ -72,11 +79,83 @@ function __upload_wav_file
     done
 }
 
+set_pga_to_unity_gain()
+{
+    tmp=$(amixer controls | grep "$1" | grep Volume)
```

Comment:
As you use very generic variable names, this first line will avoid any collision:
```
local tmp search cname scale ...
```